src/plone/restapi/services/content/configure.zcml

```diff
@@ -70,6 +70,45 @@
     permission="zope2.View"
     />
 
+  <plone:service
+    method="OPTIONS"
+    name="@upload"
```

Comment:
@tisto Hmm... I remember that we were talking about two endpoint names but I don't see any need for two differently named endpoints in the TUS implementation.